### ISSUE TITLE
docs: 'Getting started' docs fail to specify @next in installation instructions

### ISSUE THREAD

**Issue by birjj** [intlify/eslint-plugin-vue-i18n#508]
**Tell us about your environment**

- **ESLint version: 9.2.0**
- **eslint-plugin-vue version: ^9.24.1**
- **eslint-plugin-vue-i18n version: ^2.0.0**
- **Node version: v20.12.2**

**Please show your full configuration:**

```json5
/* eslint-env node */
import pluginVue from 'eslint-plugin-vue';
import pluginVueI18n from '@intlify/eslint-plugin-vue-i18n';
import js from '@eslint/js';

export default [
  js.configs.recommended,
  ...pluginVue.configs['flat/recommended'],
  ...pluginVueI18n.configs['flat/recommended'],
];

```

**What did you do?**
I followed the ["Getting started" documentation](https://eslint-plugin-vue-i18n.intlify.dev/started.html) from the website, and ran:

```sh
npm install --save-dev eslint @intlify/eslint-plugin-vue-i18n
```

However, this installed version `^2.0.0` of `@intlify/eslint-plugin-vue-i18n`, which throws an error when running eslint using the provided configuration. A fix is to instead run `npm install --save-dev @intlify/eslint-plugin-vue-i18n@next` to install version `^3.0.0-next.11`. The documentation should be updated to install the correct version.

**What did you expect to happen?**
The plugin should work when following the documentation.

**What actually happened?**
When attempting to run eslint using the provided configuration, it fails with:

```
TypeError: pluginVueI18n.configs.flat/recommended is not iterable
    at file:///Users/me/dev/test/eslint.config.mjs?mtime=1715082875973:10:21
    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:323:24)
    at async loadFlatConfigFile (/Users/me/dev/test/node_modules/eslint/lib/eslint/eslint.js:317:21)
    at async calculateConfigArray (/Users/me/dev/test/node_modules/eslint/lib/eslint/eslint.js:396:28)
    at async ESLint.lintFiles (/Users/me/dev/test/node_modules/eslint/lib/eslint/eslint.js:815:25)
    at async Object.execute (/Users/me/dev/test/node_modules/eslint/lib/cli.js:500:23)
    at async main (/Users/me/dev/test/node_modules/eslint/bin/eslint.js:153:22)
```

**Comment by omarmir** [comment]
Thanks - this helped resolve my issue!
